proto/c4echain/cfedistributor/tx.proto

```diff
@@ -1,13 +1,63 @@
 syntax = "proto3";
 package chain4energy.c4echain.cfedistributor;
 
+import "cosmos/msg/v1/msg.proto";
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "c4echain/cfedistributor/sub_distributor.proto";
 // this line is used by starport scaffolding # proto/tx/import
 
 option go_package = "github.com/chain4energy/c4e-chain/x/cfedistributor/types";
 
 // Msg defines the Msg service.
```

Comment:
// Maybe we should add some rules about naming parameters related messages.
// If massage changes all params it should be named UpdateParams
// in other cases it should have suffix Params or Param
// It is to easily distinguish the the porpose if the message
// so here we would have
// UpdateParams
// UpdateSubDistributorParams
// UpdateSubDistributorDestinationShareParam
// UpdateSubDistributorBurnShareParam